src/core/qgsabstractcontentcache.cpp

```diff
@@ -45,15 +45,29 @@ void QgsAbstractContentCacheBase::onRemoteContentFetched( const QString &, bool
 
 bool QgsAbstractContentCacheBase::parseBase64DataUrl( const QString &path, QString *mimeType, QString *data )
 {
-  const thread_local QRegularExpression sRx( QStringLiteral( "^data:(.*/.*?);?(?:base64)?,(.*)$" ) );
+  const thread_local QRegularExpression sRx( QStringLiteral( "^data:([a-zA-Z0-9+]*\\/[a-zA-Z0-9+]*?)(?:;(base64|utf8))?,(.*)$" ) );
```

Comment:
Can we not have dash in the media type like [vnd-example](https://en.wikipedia.org/wiki/Data_URI_scheme)

Reply:
Done